packages/core/src/input/input-groups.stories.ts

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { html } from 'lit-html';
+import '@clr/core/input/register.js';
+
+export default {
+  title: 'Forms (Preview)/Input Groups/Stories',
+  component: 'cds-input-group',
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=20%3A1',
+    },
+  },
+};
+
+export const group = () => {
+  return html`
+    <div cds-layout="vertical gap:lg">
+      <cds-input-group>
+        <label>Host URL</label>
+        <cds-select cds-layout="align:shrink">
+          <label>Protocol</label>
+          <select>
+            <option>http://</option>
+            <option>https://</option>
+          </select>
+        </cds-select>
+        <cds-input>
+          <label>Host Port</label>
+          <input placeholder="localhost:8000" type="url" />
+          <cds-control-action action="suffix" readonly aria-label="host status stable" title="host status stable">
+            <cds-icon shape="cloud" badge="success"></cds-icon>
+          </cds-control-action>
```

Comment:
Control actions are similar to alert actions. They are by default a button type that allows us to add interactive buttons as a prefix or suffix to a given component. Control actions can also be set as readonly for decorative icons or prefix/suffix text labels.